CHANGELOG/CHANGELOG-6.3.md

```diff
@@ -0,0 +1,399 @@
+# Release notes for v6.3.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v6.2.0
+
+## Changes by Kind
+
+### API Change
+
+- Add VolumeGroupSnapshot API definitions ([#814](https://github.com/kubernetes-csi/external-snapshotter/pull/814), [@RaunakShah](https://github.com/RaunakShah))
+
+### Feature
+
+- Add a marker to the snapshot-controller manifests to enable feature gates in CSI prow jobs. ([#790](https://github.com/kubernetes-csi/external-snapshotter/pull/790), [@RaunakShah](https://github.com/RaunakShah))
+- Create Volume functionality for volume group snapshots ([#826](https://github.com/kubernetes-csi/external-snapshotter/pull/826), [@RaunakShah](https://github.com/RaunakShah))
```

Comment:
Do we need to update README as well this? Is there a k8s version requirement?

Reply:
The feature is not complete yet.  CreateVolumeGroup is implemented but DeleteVolumeGroup is not (there is a WIP PR but it is not complete).  I temporarily disabled the feature to prevent people from using it.  I have the following item in the changelog that disabled the feature.  Let me know if I should call this out more clearly.
```
- Disable the enable-volume-group-snapshots feature flag. ([#901](https://github.com/kubernetes-csi/external-snapshotter/pull/901), [@xing-yang](https://github.com/xing-yang))
```

Reply:
Should I say something like this?
```
Create Volume functionality for volume group snapshots (Note: Delete is not implemented yet so this feature is not complete).
```

Reply:
Maybe we should remove any notes about volumegroup snapshot until the feature is usable.

Reply:
Sure.  That's probably better.

Reply:
I kept the removed ones in a different commit: https://github.com/kubernetes-csi/external-snapshotter/pull/908/commits/de93167de3a89b56493c1a826c197f46a4c872dd